### ISSUE TITLE
module-getter rule false positive

### ISSUE THREAD

**Issue by keirlawson** [Gillespie59/eslint-plugin-angular#240]
This rule is getting triggered for the following code:

```js
const MODULE_NAME = 'mymodule';

angular
  .module(MODULE_NAME, [])
  .service('backendService', BackendService);
```

**Comment by Gillespie59** [comment]
Thanks @keirlawson 

Could you send a PR with a failing unit test ? ;)

Manu

**Comment by vrockai** [comment]
There seem to be another false positive for this rule, should I create a new issue for that? It happens when an object has the `config` method, like:
```
/** @ngInject */
function lazyLoadDebugConfig($ocLazyLoadProvider) { 
  $ocLazyLoadProvider.config({
    debug: false
  });
}
```

The ESLint errors than go like this:
```
  3:1  error  Avoid using a variable and instead use chaining with the getter syntax  angular/module-getter
  3:1  error  You should use the function syntax for DI                               angular/di

```

**Comment by kuhnroyal** [comment]
Same here:

```javascript
$localForageProvider.config({
    driver: 'localStorageWrapper',
    name: 'test',
    storeName: 'test'
});
```

**Comment by Gillespie59** [comment]
I solved the first issue. But the last 2, I do not know how we can solve it. 
The problem is the `config` method. We can not detect that your providers is not an angular module easily.

**Comment by Gillespie59** [comment]
I close this issue. See #223
